src/vector/index.html

```diff
@@ -60,7 +60,7 @@
   </head>
   <body style="height: 100%; margin: 0;">
     <noscript>Sorry, Element requires JavaScript to be enabled.</noscript> <!-- TODO: Translate this? -->
-    <section id="matrixchat" style="height: 100%;" class="notranslate"></section>
+    <section id="matrixchat" style="height: 100%; display: flex; flex-direction: column;" class="notranslate"></section>
```

Comment:
This would not be needed once moving to the new wrapper suggestion

```suggestion
    <section id="matrixchat" style="height: 100%;" class="notranslate"></section>
```

Reply:
The change is removed as not needed, thanks!